snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -124,78 +135,69 @@ instances:
     #
     # retries: 5
 
-    ## @param tags - list of key:value element - optional
-    ## List of tags to attach to every metric, event and service check emitted by this integration.
-    ##
-    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
-    #
-    # tags:
-    #   - <KEY_1>:<VALUE_1>
-    #   - <KEY_2>:<VALUE_2>
-
     ## @param loader - string - optional - default: python
     ## Check loader to use. Available loaders:
-    ## - core: (recommended) will use new corecheck SNMP integration
-    ## - python: will use legacy python SNMP integration
+    ## - core: (recommended) Uses new corecheck SNMP integration
+    ## - python: Uses legacy python SNMP integration
     #
-    # loader: core
+    # loader: python
```

Comment:
Should be `loader: core`, that's what we want to recommend to users.

Reply:
Hey @ofek , we want for this `loader: core` to be uncommented, but not mark it as required is that possible ?

If not, shall we add an option to make it possible ?

An alternative is to make it `required` as workaround.

Reply:
Never mind. seems possible with `enabled` :)